packages/amplify_authenticator/lib/src/widgets/form_field.dart

```diff
@@ -216,11 +216,14 @@ abstract class AuthenticatorFormFieldState<FieldType, FieldValue,
   }
 
   /// Margin below the form field
-  EdgeInsetsGeometry? get marginBottom => FormFieldConstants.marginBottom;
+  double? get marginBottom => FormFieldConstants.marginBottom;
 
   /// Gap between the label and the form field
   double? get labelGap => FormFieldConstants.gap;
 
+  /// Widget to show above the label.
+  Widget? get surlabel => null;
```

Comment:
Open to other names here. The username selection widget was previously set with `suffix`, but since it has a new placement, I added this.

This widget is above the label. "sur" as a prefix I think typically means "above" in more of a figurative sense (surcharge), but can also be in the literal sense (a surtitle is a like a subtitle but is placed at the top)

Reply:
Sounds good - sur means above/on in French, e.g. *sur la table*